legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -96,7 +96,7 @@ def get_ia_revision(filing, business) -> dict:
         party_email = ''
         for party in filing.json['filing']['incorporationApplication']['parties']:
             if next((x for x in party['roles'] if x['roleType'] == 'Completing Party'), None):
-                party_email = party['email']
+                party_email = party['officer']['email']
```

Comment:
To avoid issues like this it is a good practice to use something like this
party.get('officer',{}).get('email', None)

Reply:
ok